src/main/java/com/staff_management_system/com/database_connection_package/DatabaseConnect.java

```diff
@@ -0,0 +1,51 @@
+package com.database_connection_package;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class DatabaseConnect {
+    /**
+     * Connection object to open port to db
+     */
+    private final ThreadLocal<Connection> aConnection = new ThreadLocal<>();
+
+    /**
+     * closes the database connection
+     */
+    public void terminate() {
+        try {
+            aConnection.get().close();
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+    }
+
+    /**
+     * establishes the database connection
+     *
+     * @return Connection to the database
+     */
+    public Connection initialize() {
+        try {
+            Class.forName("org.postgresql.Driver"); // loads the JDBC Driver for PostGreSQL
+
+            //Database location
+            String url = "jdbc:postgresql://127.0.0.1:5432";

```

Comment:
url, username and password should be constructor parameter .. so that the main logic class can create connection to "any database" and not only the database you have created now. This is how you make your code resuable.

Reply:
* reusable

Reply:
Will change it.